.gitignore

```diff
@@ -13,3 +13,4 @@ files/*.tsv
 Gemfile.lock
 scratch/
 .vendor
+.bundle
```

Comment:
Not sure if this is a pertinent change or not. Doesn't appear to relate to the new snippet library.

Reply:
When you build the site locally it creates a `.bundle` directory in the root, we shouldn't track this directory so I added it to the ignore list. I'll open a separate PR for this.

Reply:
https://github.com/carpentries-incubator/hpc-intro/pull/313